conduct/conduct_reporting_online.md

```diff
@@ -0,0 +1,80 @@
+information:
+
+* Your contact info (not required, if you would like to remain anonymous please
+  use the form)
+* Names (real, nicknames, or pseudonyms) of any individuals involved. If there
+  were other witnesses besides you, try to include them as well
+* When and where the incident occurred. Please be as specific as possible.
+* Your account of what occurred. If there is a publicly available record (e.g. a
+  mailing list archive or GitHub/gitter.im) please include a link
+* Any extra context you believe exists for the incident
+* If you believe this incident is ongoing
+* Any other information you believe we should have
+
+
+## What happens after you file a report?
+
+You will receive an email from the Project Jupyter Code of Conduct Committee
+acknowledging receipt immediately. We promise to acknowledge receipt within 24
+hours (and will aim for much quicker than that).
+
+The working group will immediately meet to review the incident and determine:
+* What happened
+* Whether this event constitutes a Code of Conduct violation
+* Who the bad actor was
+* Whether this is an ongoing situation, or if there is a threat to anyone's
+  physical safety
+
+
+If this is determined to be an ongoing incident or a threat to physical safety,
+the committee's immediate priority will be to protect everyone involved. This
+means we may delay an "official" response until we believe that the situation
+has ended and that everyone is physically safe.
+
+Once the Code of Conduct Committee has a complete account of the events they
+will make a decision as to how to respond.
+
+Responses may include:
+
+* No action (if we determine no violation occurred)
+* A private reprimand from the Code of Conduct Committee to the individual(s)
+  involved
+* A public reprimand
+* An imposed vacation (for example: asking someone to "take a week off" from a
+  mailing list, GitHub, etc.)
+* A temporary or permanent ban from some or all Jupyter spaces
+* A request for a public or private apology
+* We'll respond within one week to the person who filed the report with either a
+  resolution or an explanation of why the situation is not yet resolved
+
+Once we've determined our final action, we'll contact the original reporter to
+let them know what action (if any) we'll be taking. We'll take into account
+feedback from the reporter on the appropriateness of our response, but we don't
+guarantee we'll act on it.
+
+Finally, the Code of Conduct Committee will make a report on the situation to
+the Jupyter Steering Council. The council may choose to publish a report of the
+incident.
+
+## Appealing the Code of Conduct Committee’s Response
+
+Only permanent resolutions (such as bans) may be appealed. To appeal a decision
+of the Code of Conduct Committee, contact the Jupyter Steering Council at
+[*steeringcouncil@jupyter.org*](mailto:steeringcouncil@jupyter.org) with your
+appeal and they will review the case.
```

Comment:
Added a bit of info about timeframe and new information, though this could probably be improved further over time.  Don't have a lot of ideas right now...